README.md

```diff
@@ -61,3 +61,7 @@ the binary after compilation.
 Future of this crate
 --------------------
 There are currently [open issues](https://github.com/rust-lang/cargo/issues/3670) upstream in cargo which might make this crate redundant in the future.
+
+Contributors
+------------
+This crate would be not be what it is today without the many contributions of [@barskern](https://github.com/barskern)!
```

Comment:
There is a typo btw; "would **be** not be what".

Reply:
Arrrrg! Fixed! Also, I guess its "contributions by..."